### DISCUSSION TITLE
Ruby changes from JLReq to jlreq-d

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#57] [General]
JLReqからjlreq-dで何点かルビの要件が変わりますが、ここにリストしたく。下に挙げた例以外にありますか？また補足していただけるとありがたいです。

- 親文字もしくはルビが長い場合（例えば4全角以上？）：折り返しても良い。googleが実験に成功。セグメント化されている場合（熟語ルビ）、セグメントの区切れを尊重する。
- 親文字に比べてルビの長い場合：ルビではなく、行間注の扱いとする。
- モノルビ、グループルビ、熟語ルビの用語は紹介するが機能の記述には使わない。（が、記述方法の変更であって、これによって要件が変化するわけではない）

**Comment by murata2makoto** [comment]
両側ルビについてはJLReqのときは「このように付ける例も非常に少ない」と書かれていました。JLReq-dでは目視確認による調整はないので、両側ルビのときは前の行の下（左）側のルビと今の行の上（右）側ルビが衝突する可能性があります（広すぎる行間を確保するか、行間が一定しないことを許容しない限り）。こういうことを追加するのはどうでしょう。

**Reply by kidayasuo** [reply]
先日のミーティングでも議論になりましたね。衝突防止や行の折り返しの処理が複雑で、印刷では手動の調整に頼っていたこと、自動化しにくいこと、プライオリティが低いこと、からjlreq-dでは落としたというようなコメントを、サイドノートに書いきましょう。また小林さんの提案された議論セクションに残すと良さそうですかね。
